helm_helpers.go

```diff
@@ -506,8 +506,8 @@ func deployTiller(namespace string, serviceAccount string, defaultServiceAccount
 // initHelmClientOnly initializes the helm client only (without deploying Tiller)
 func initHelmClientOnly() (bool, string) {
 	cmd := command{
-		Cmd:         "bash",
-		Args:        []string{"-c", "helm init --client-only "},
+		Cmd:         "helm",
+		Args:        []string{"init", "--client-only"},
```

Comment:
There used to be a space after `--client-only` (`"--client-only"` instead of `"--client-only "`). Not sure what the impact is, might be a typo that this PR fixes now.